src/styles/RepositoryAvatar.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import {borderRadius} from "./variables";
+
+const RepositoryAvatar = styled.img`
+  margin-right: 10px !important;
+  margin-bottom: -10px;
```

Comment:
```suggestion
  margin-right: ${gutter.gutter} !important;
  margin-bottom: -${gutter.gutter};
```

Use `${gutter.gutter}`

Reply:
Done.